### ISSUE TITLE
UI: (13.C) Overview Top 50 Domains Insufficient Contrast (CRASM-369)

### ISSUE THREAD

**Issue by hawkishpolicy** [cisagov/XFD#427]
# 🐛 Summary #

Insufficient contrast is provided for “*Top 50 domains with open vulnerabilities” text. The color difference of text and its background should have a contrast ratio of at least 4.5:1. Using contrast analyzers, spot test various text and backgrounds throughout the site or application to ensure a contrast ratio of at least 4.5:1. When insufficient contrast issues are found, adjust either one or the other color to meet a minimum of 4.5:1 ratio.

## Any helpful log output or screenshots ##

![image](https://github.com/cisagov/XFD/assets/123108455/09d17eb0-b715-43f5-a304-db75513c398e)
